utils/predictions.py

```diff
@@ -60,6 +60,7 @@ def get_uri(self, **kwargs):
          """
 
         self._uri = QgsDataSourceURI()
+        print(self._uri)
```

Comment:
Better remove print statements. In some cases this case troubles in QGIS plugins